plugins/analytics-module-ga/README.md

```diff
@@ -0,0 +1,131 @@
+# Analytics Module: Google Analytics
+
+This plugin provides an opinionated implementation of the Backstage Analytics
+API for Google Analytics. Once installed and configured, analytics events will
+be sent to GA as your users navigate and use your Backstage instance.
+
+This plugin contains no (and will never contain any) other functionality.
+
+## Installation
+
+1. Install this plugin in your Backstage instance: `yarn add @backstage/plugin-analytics-module-ga`
```

Comment:
You could add that this should be installed at the `/packages/app/package.json` level and not the root level `/package.json`.

Reply:
Good call.